File: apps/docs/src/code/python/basics/process_stop.py
from e2b import Sandbox

session = Sandbox.create(id="Nodejs")

npm_init = session.process.start("npm init -y")
npm_init.kill() # $HighlightLine

# There will be no output because we immediately kill the `npm_init` process
print(npm_init.stdout)

session.close()

File: apps/docs/src/code/js/basics/process_stop.js
import { Sandbox } from '@e2b/sdk'

const sandbox = await Sandbox.create({
  id: 'base',
})

const npmInit = await sandbox.process.start({
  cmd: 'npm init -y',
})
await npmInit.kill() // $HighlightLine
// There will be no output because we immediately kill the `npm_init` process
console.log(npmInit.output.stdout)

await sandbox.close()
